homeassistant/components/xiaomi_miio/switch.py

```diff
@@ -288,15 +243,17 @@ async def _try_command(self, mask_error, func, *args, **kwargs):
 
     async def async_turn_on(self, **kwargs):
         """Turn the plug on."""
-        result = await self._try_command("Turning the plug on failed.", self._plug.on)
+        result = await self._try_command("Turning the plug on failed.", self._device.on)
 
         if result:
             self._state = True
             self._skip_update = True
 
     async def async_turn_off(self, **kwargs):
         """Turn the plug off."""
-        result = await self._try_command("Turning the plug off failed.", self._plug.off)
+        result = await self._try_command(
+            "Turning the plug off failed.", self._device.off
```

Comment:
We don't end logging messages with period.

Reply:
Thanks, I fixed it throughout the switch.py file.